craco.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   webpack: {
     configure: (webpackConfig) => {
       const jsonLinesLoader = {
-        test: require.resolve("./src/data.json"),
+        test: /\.jsonl$/,
```

Comment:
Why did we switch to `.jsonl` files explicitly here? Did @colincadams approve that change previously and I just missed it?

Reply:
No, I just didn't know that we can't use `.jsonl` for now. Switched to previous method.